WPMedia/arc-themes-components
# THEMES-1486: Add icons to component library

## Ticket

- [THEMES-1486](https://arcpublishing.atlassian.net/browse/THEMES-1486)

## Description

This PR adds a missing icon, the check mark, and new social media icons (Google, TikTok).

## Acceptance Criteria

- Check mark icon is available.
- Google and TikTok icons are available (2 variants each)

## Test Steps

1. Checkout branch - `git checkout THEMES-1486`
2. This can be tested by adding a simple feature to your local feature pack.
3. Add `components/features/test/default.jsx` with the following:
```
import { Icon } from "@wpmedia/arc-themes-components";

const TestIcons = () => (
  <>
    <Icon name="Check" />
    <Icon name="Google" />
    <Icon name="GoogleColor" />
    <Icon name="TikTok" />
    <Icon name="TikTokColor" />
  </>
);

export default TestIcons;
```
4. Start fusion with `npx fusion start`.
5. Add the test block to a page and verify the icons are showing.

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1486]: https://arcpublishing.atlassian.net/browse/THEMES-1486?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
